bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/i18n/gardena_de.properties

```diff
@@ -38,10 +38,8 @@ thing-type.config.gardena.account.apiKey.label = API Schlüssel
 thing-type.config.gardena.account.apiKey.description = Der Gardena Smart System Integrations API-Schlüssel
 thing-type.config.gardena.account.connectionTimeout.label = Verbindungszeitüberschreitung
 thing-type.config.gardena.account.connectionTimeout.description = Der Timeout in Sekunden für Verbindungen zur Gardena Smart System Integrations API
-thing-type.config.gardena.account.email.label = E-Mail
-thing-type.config.gardena.account.email.description = E-Mail-Adresse für die Anmeldung im Gardena Smart System
-thing-type.config.gardena.account.password.label = Passwort
-thing-type.config.gardena.account.password.description = Passwort zur Anmeldung im Gardena Smart System
+thing-type.config.gardena.account.apiSecret.label = API Secret
+thing-type.config.gardena.account.apiSecret.description = Das Gardena Smart System Integrations API-Secret
```

Comment:
Please revert the changes for the German translation and provide it in Crowdin when the PR is merged. For reference:
https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations